src/table/settings/economy_settings.ini

```diff
@@ -280,3 +280,15 @@ str      = STR_CONFIG_SETTING_INFRASTRUCTURE_MAINTENANCE
 strhelp  = STR_CONFIG_SETTING_INFRASTRUCTURE_MAINTENANCE_HELPTEXT
 post_cb  = [](auto) { InvalidateWindowClassesData(WC_COMPANY_INFRASTRUCTURE); }
 cat      = SC_BASIC
+
+[SDT_VAR]
+var      = economy.timekeeping_units
+type     = SLE_UINT8
+flags    = SF_GUI_DROPDOWN | SF_NEWGAME_ONLY | SF_SCENEDIT_TOO
+def      = 0
+min      = 0
+max      = 1
```

Comment:
In extension to enum, please check with other settings, but please make this strings. I really dislike settings that say: `timekeeping_units = 0`. Much better would be: `timekeeping_units = wallclock`.

We have done this for many settings already, but not all. As converting sucks. But for new ones it is easy :)